client/src/locales/en.json

```diff
@@ -103,6 +103,9 @@
   "authorization.reset-password.button-content": "Reset password",
   "authorization.actions.reset": "An email with instructions to reset password has been sent to {data} if it is associated with an EOC account. Your existing password has not been changed.",
   "authorization.actions.reset-password-not-active-account": "Account: {data} not active yet. Please go to your email inbox and activate account.",
+  "authorization.password-recovery-success": "Password successfully updated. Please go to the {link} and sign in to app.",
+  "authorization.pass-recovery-form.heading": "Set new password",
+  "authorization.pass-recovery-form.validation-error": "Passwords validation went wrong. Please try again.",
```

Comment:
when this error is displayed - it sounds weird

Reply:
Any proposals?

Reply:
Any answer to my question - when this error is displayed?

Reply:
when the user omits somehow front validation and the server sends his own validation error to front.

Reply:
if such scenario takes place then server should just send bad request response

Reply:
It's removed. 
Now will be just Bad Request and common error message